configure

```diff
@@ -1153,7 +1153,6 @@ for opt do
   *)
       echo "ERROR: unknown option $opt"
       echo "Try '$0 --help' for more information"
-      exit 1
```

Comment:
It is wrong to go through unknown configure options and ignore them.

Reply:
unknown options are not ignores, error message is still there.
I'd like to have a script compatible between different qemu versions

Reply:
take a look to commit message

Reply:
If you want a script to work with different qemu versions, please customize your script to give different configure options based on different qemu versions in stead.

Reply:
There is a `VERSION` file in the top directory for your convenience.